orm/utils.go

```diff
@@ -181,18 +181,36 @@ func ToInt64(value interface{}) (d int64) {
 	return
 }
 
-// snake string, XxYy to xx_yy
+// snake string, XxYy to xx_yy , XxYY to xx_yy
```

Comment:
I can't think anyone use that column name `n_o` or `i_d`  . like `URL` will set tag `column(URL)`